crates/fuzzing/src/generators/config.rs

```diff
@@ -349,6 +349,11 @@ impl<'a> Arbitrary<'a> for Config {
                 }
             };
 
+            // Don't allow too many linear memories per instance since massive
+            // virtual mappings can fail to get allocated.
+            cfg.min_memories = cfg.min_memories.min(10);
```

Comment:
Shouldn't `min` be 0 or 1?

Reply:
The maximum/minimum being bigger than 1 indicates the multi-memory proposal is active, if that's your question? I wanted to leave that part of fuzzing active ideally.

Reply:
Yeah, but I would have thought we would have also wanted to generate modules that may between 0 and 10 memories when we use the pooling allocator just to check that things don't break if, e.g., no memories are defined?

Reply:
This style of using `min()` always confuses me, and I think it's confusing you too. It's setting `min_memories` to whichever is smaller of 10 or the randomly-chosen value, which means that it's at _most_ 10.

Reply:
Ah! Ok, sorry, didn't see it right. There's a previously generated random value in `min_memories` that we are trying to constrain. Please ignore me!